data/hai/hai reveal 5 philosophers.txt

```diff
@@ -753,16 +734,16 @@ mission "Hai Rescue [C04] Foundry"
+		log `Equipped eleven Unfettered Hai Shield Beetles, led by Teeneep and General Choot'k, for the battle at Philosopher's Paradise.`
 		log `People` `General Choot'k` `Supreme commander of the Unfettered Hai military. It is unclear whether he also leads the Unfettered Hai government. Compared to his troops, he is a man of great restraint and intelligence. He's also unusually large for a Hai, taller than most humans.`
 		conversation
-			`Teeneep and the Unfettered leader leave a Shield Beetle together. Teeneep is still wearing her fencing outfit and swords, but she now wears a sash that indicates she's a Hai Elder. She's cleaned off most of the blood stains, but left just enough to remind people she can fight. Marshal Kaine approaches from the opposite direction.`
-			`	The Unfettered leader opens his mouth to speak, but Teeneep raises her hand to silence him. You're surprised to see that he stops talking without complaint. Teeneep must have some hold over him.`
-			`	"This is General Choot'k," Teeneep says, "supreme commander of the Unfettered Hai military. I will join him on his flagship during this fight. Ten of our Shield Beetles must be re-outfitted. His is already equipped with superior equipment from past conquests."`
-			`	Marshal Kaine responds, "We are preparing a group of 200 elite operatives, trained in fighting pirates. I will leave it to Captain <last> to see them safely to the surface. We have no transport ships that can survive such a battle. If they reach Philosopher's Paradise, they will free your people."`
+			`Teeneep and the Unfettered leader leave a Shield Beetle together. Teeneep is still wearing her fencing outfit and swords, but she now wears a sash that indicates she's a Hai Elder. She's patched up the tears, but left just enough to remind people she can fight. The marshal, a man named Kaine, approaches from the opposite direction.`
+			`	The Unfettered leader opens his mouth to speak, but Teeneep raises her hand to silence him and steps forward instead. Surprisingly, he stops without complaint.`
+			`	"This is General Choot'k, supreme commander of the Unfettered Hai military. I will join him on his flagship during this fight. Ten of our Shield Beetles must be re-outfitted. His is already equipped with superior equipment from past conquests."`
+			`	Marshal Kaine responds, "Good to meet you. We're currently preparing a group of 200 elite operatives, trained in fighting pirates. I will leave it to Captain <last> to see them safely to the surfac, since we have no transport ships that can survive a battle of this scale. If they reach Philosopher's Paradise, they will free your people."`
```

Comment:
```suggestion
			`	Marshal Kaine responds, "Good to meet you. We're currently preparing a group of 200 elite operatives, trained in fighting pirates. I will leave it to Captain <last> to see them safely to the surface, since we have no transport ships that can survive a battle of this scale. If they reach Philosopher's Paradise, they will free your people."`
```